test/mindwendel_web/live/admin/brainstorming_live/edit_test.exs

```diff
@@ -164,16 +165,16 @@ defmodule MindwendelWeb.Admin.BrainstormingLive.EditTest do
         live(conn, ~p"/admin/brainstormings/#{brainstorming.admin_url_id}/edit")
 
       # reload brainstorming to check for changes:
-      brainstorming = Brainstormings.get_brainstorming!(brainstorming.id)
-      assert Enum.count(brainstorming.lanes) == 1
+      lanes = Lanes.get_lanes_for_brainstorming(brainstorming.id)
+      assert Enum.count(lanes) == 1
 
       edit_live_view
       |> element("button", "Empty")
       |> render_click()
 
       # reload brainstorming to check for changes:
-      brainstorming = Brainstormings.get_brainstorming!(brainstorming.id)
-      assert Enum.empty?(brainstorming.lanes)
+      lanes = Lanes.get_lanes_for_brainstorming(brainstorming.id)
+      assert Enum.empty?(lanes)
```

Comment:
`Enum.empty?` works but I think error messages are better with `assert lanes == []` (the first one just says expected truthy got flasy while the second one shows you the contents of the list iirc)